db/migrate/20220520211359_create_likes.rb

```diff
@@ -0,0 +1,9 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.string :name
+
```

Comment:
The likes table should have column name `author_id` as a foreign key, and `post_id` as a foreign key.